docs/quickstarts/sso-quickstart.md

```diff
@@ -126,6 +133,46 @@ At this point, your add-in project has been created and contains the code that's
 
     ![User profile information in Excel worksheet](../images/sso-user-profile-info-excel.png)
 
+### Outlook
+
+If you've created an Outlook add-in, complete the following steps to try it out.
+
+> [!NOTE]
+> If you've created an Excel, Word, or PowerPoint add-in, complete the steps in the previous [Excel, Word, and PowerPoint](#excel-word-and-powerpoint) section instead.
+
+1. When the SSO configuration process completes, run the following command to build the project and start the local web server.
+
+    > [!NOTE]
+    > Office Add-ins should use HTTPS, not HTTP, even when you are developing. If you are prompted to install a certificate after you run the following command, accept the prompt to install the certificate that the Yeoman generator provides.
+
+    ```command&nbsp;line
+    npm start
+    ```
+
+2. Follow the instructions in [Sideload Outlook add-ins for testing](/outlook/add-ins/sideload-outlook-add-ins-for-testing) to sideload the add-in in Outlook. Make sure that you're signed in to Outlook with a user that's a member of the same Office 365 organization as the Office 365 administrator account that you used to connect to Azure while configuring SSO in step 3 of the [previous section](#configure-sso). Doing so establishes the appropriate conditions for SSO to succeed. 
+
+3. In Outlook, compose a new message.
+
+4. In the message compose window, choose the **Show Taskpane** button in the ribbon to open the add-in task pane.
+
+    ![Outlook add-in button](../images/outlook-sso-ribbon-button.png)
+
+5. At the bottom of the task pane, choose the **Get My User Profile Information** button to initiate the SSO process. 
+
+    > [!NOTE] 
+    > If you're not already signed in to Office at this point, you'll be prompted to sign in. As described previously, you should sign in with a user that's a member of the same Office 365 organization as the Office 365 administrator account that you used to connect to Azure while configuring SSO in step 3 of the [previous section](#configure-sso), if you want SSO to succeed.
```

Comment:
It seems like this sign in information is a repeat of step 2. Is there a distinction I'm missing?

Reply:
Fair point -- it is basically a duplicate of info mentioned previously; I'll remove it.